internal/resources/metal/connection/resource.go

```diff
@@ -4,17 +4,15 @@ import (
 	"context"
 	"fmt"
 	"reflect"
-	"strconv"
 
+	"github.com/equinix/equinix-sdk-go/services/metalv1"
 	equinix_errors "github.com/equinix/terraform-provider-equinix/internal/errors"
 	"github.com/equinix/terraform-provider-equinix/internal/framework"
 	"github.com/hashicorp/terraform-plugin-framework/diag"
 	"github.com/hashicorp/terraform-plugin-framework/path"
 	"github.com/hashicorp/terraform-plugin-framework/resource"
 	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
-	"github.com/hashicorp/terraform-plugin-framework/types"
 	"github.com/packethost/packngo"
```

Comment:
It looks like there are still some things in this file and (possibly other files in this package) that are still using `packngo`.  It appears to be working fine, so this may be more of a nit.

Reply:
oof, good catch. all fixed